build.sbt

```diff
@@ -13,20 +12,21 @@ lazy val microservice = (project in file("."))
   .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(playSettings: _*)
   .settings(scalaSettings: _*)
```

Comment:
These can be removed along with [line](https://github.com/hmrc/binding-tariff-ruling-frontend/blob/main/build.sbt#L9) `lazy val playSettings: Seq[Setting[_]] = Seq.empty`